stingray/lightcurve.py

```diff
@@ -56,30 +68,36 @@ def __init__(self, time, counts, input_counts=True):
             The start time of the light curve.
 
         """
-
         assert np.all(np.isfinite(time)), "There are inf or NaN values in " \
                                           "your time array!"
-
         assert np.all(np.isfinite(counts)), "There are inf or NaN values in " \
                                             "your counts array!"
+        if err is not False:
+            assert np.all(np.isfinite(err)), "There are inf or NaN " \
+                                             "values in your err array!"
+        else:
+            err = np.sqrt(np.asarray(counts))
```

Comment:
I think we should add another layer of refinement when the counts per bin are very small (counts  < ~20-30)
`error = 1 + sqrt(N + 0.75)`  (the Gehrels approximation [link](http://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1986ApJ...303..336G&amp;data_type=PDF_HIGH&amp;whole_paper=YES&amp;type=PRINTER&amp;filetype=.pdf), [link2](http://stsdas.stsci.edu/cgi-bin/gethelp.cgi?explain_errors) )